lib/vagrant-omnibus/action/install_chef.rb

```diff
@@ -214,7 +214,7 @@ def fetch_or_create_install_script(env)
                   set version=%1
                   set dest=%~dp0chef-client-%version%-1.windows.msi
                   echo Downloading Chef %version% for Windows...
-                  powershell -command "(New-Object System.Net.WebClient).DownloadFile('#{url}?v=%version%', '%dest%')"
+                  powershell -command "if(test-path '#{url}/chef-client*%version%*.msi'){[array]$tempstore = dir #{url} | ?{$_.name -like 'chef-client*%version%*.msi'};copy $tempstore[0].fullname %dest%}else{(New-Object System.Net.WebClient).DownloadFile('#{url}?v=%version%', '%dest%')}"
```

Comment:
URI exactly what I was looking for. I figured there is a single construct that allows for both. Big thanks, I'll play with it and amend the pull request.